src/Framework/AspNetCoreAnalyzers/src/Analyzers/Http/RequestDelegateReturnTypeAnalyzer.cs

```diff
@@ -59,7 +59,10 @@ public override void Initialize(AnalysisContext context)
                             var resolvedOperation = WalkDownConversion(returnedValue);
                             var returnType = resolvedOperation.Type;
```

Comment:
it is still failing, maybe this will fix it:
```suggestion
                            var returnType = resolvedOperation?.Type;
```